### PR TITLE
Feature/updates gitignore

### PR DESCRIPTION
**Pull request by alejouribesanchez**
- Add .gitignore file
- Add clean up for too long lines warnings
- Add updated version for user agent
- Add param document_id description for update_document function

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+.idea/
```
Comment (manycoding) [inline .gitignore:2]: Use global gitignore instead, no need to add extra files

File: veryfi/__init__.py [hunk 2]
```diff
@@ -215,14 +215,17 @@ def process_document_url(
...
-        :param file_url: Required if file_urls isn't specified. Publicly accessible URL to a file, e.g. "https://cdn.example.com/receipt.jpg"
-        :param file_urls: Required if file_url isn't specifies. List of publicly accessible URLs to multiple files, e.g. ["https://cdn.example.com/receipt1.jpg", "https://cdn.example.com/receipt2.jpg"]
-        :param categories: List of categories to use when categorizing the document
-        :param delete_after_processing: Delete this document from Veryfi after data has been extracted
-        :param max_pages_to_process: When sending a long document to Veryfi for processing, this paremeter controls how many pages of the document will be read and processed, starting from page 1.
-        :param boost_mode: Flag that tells Veryfi whether boost mode should be enabled. When set to 1, Veryfi will skip data enrichment steps, but will process the document faster. Default value for this flag is 0
-        :param external_id: Optional custom document identifier. Use this if you would like to assign your own ID to documents
+        Process Document from url and extract all the fields from it :param file_url: Required if file_urls isn't
```
Comment (manycoding) [inline veryfi/__init__.py:218]: Formatting is off, should be a :param per line

**Review by manycoding — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (alejouribesanchez) [reply .gitignore:2]: Where is the global gitignore?

File: veryfi/__init__.py (hunk 2 above)
Comment (alejouribesanchez) [reply veryfi/__init__.py:218]: got it

File: .gitignore (hunk 1 above)
Comment (alejouribesanchez) [reply .gitignore]: I removed local's project gitignore to use global on my machine

**Review by manycoding — APPROVED** [review]

**Pull request merged**